tests/runtime/basic

```diff
@@ -63,6 +63,13 @@ EXPECT_REGEX [ ]+[a-zA-Z_\*\s]+
 REQUIRES_FEATURE btf
 TIMEOUT 1
 
+NAME it doesn't list the first rawtracepoint param or retval
```

Comment:
Not sure I understand. What is the full expected output here?

Reply:
I didn't want to rely on the exact param output in case the API changes so instead I'm just asserting that the retval and void * (first param) are not listed.
Before this change this is what you would see:
```
rawtracepoint:vmlinux:consume_skb
    struct sk_buff * skb
    void * location
    void * __data
    struct sk_buff * skb
    void * location
    int retval
```
What you want to see is this:
```
rawtracepoint:vmlinux:consume_skb
    struct sk_buff * skb
    void * location
```

Reply:
It should probably go to the comment for this test case in some form.